packages/@n8n/nodes-langchain/nodes/chains/TextClassifier/TextClassifier.node.ts

```diff
@@ -227,13 +234,27 @@ ${fallbackPrompt}`,
 			const prompt = ChatPromptTemplate.fromMessages(messages);
 			const chain = prompt.pipe(llm).pipe(parser).withConfig(getTracingConfig(this));
 
-			const output = await chain.invoke(messages);
-			categories.forEach((cat, idx) => {
-				if (output[cat.category]) returnData[idx].push(items[itemIdx]);
-			});
-			if (fallback === 'other' && output.fallback)
-				returnData[returnData.length - 1].push(items[itemIdx]);
+			try {
+				const output = await chain.invoke(messages);
+
+				categories.forEach((cat, idx) => {
+					if (output[cat.category]) returnData[idx].push(item);
+				});
+				if (fallback === 'other' && output.fallback) returnData[returnData.length - 1].push(item);
+			} catch (error) {
+				if (this.continueOnFail(error)) {
+					returnData[0].push({
```

Comment:
So in case of multiple outputs + continueOnFail we need to put the error into the first output?

Reply:
No, not necessarily, it'll take the errors from any output and put them in the error-output. However, if there's an error, we don't know where to put it, so I put it on the first output. Should we have different behavior here, you think?

Reply:
ah, I see. I guess it's fine then. Changing this to e.g. returnData as an object `{ outputs, error }` would be more clear but will break way too much of the existing code, not practical.